report-jtreg/src/main/resources/io/jenkins/plugins/report/jtreg/BuildReportExtendedPlugin/index.jelly

```diff
@@ -70,7 +70,9 @@
                                                 document.getElementById('button_expand_${suitesStatus.count}_${testStatus.count}').style.display = 'inline';
                                                 return false;">(collapse)</a>
                                         <j:if test="${it.isDiffTool()}">
-                                          <a target="_blank" href="${it.getTrackingUrl(t)}" style="font-size: smaller;" >(track)</a>
```

Comment:
Maybe I overloked, but it seems the getTrackingUrl method is still theere. Please drop it (and aaybe also other no longer used ones becasue it.)

Maybe add the track to the exmaple?

One more example should be call to the trace diff with this particualr test... (as it is heavily used and everybody will probably need to write this to config)

Reply:
ok, i will look at that

Reply:
Please do not forget this. This is a bit blocking our depoyment.

Reply:
@patrikcerbak  ^

Reply:
ty!